### DISCUSSION TITLE
Dark Theme for Ghidra still Not Avalible ?

### DISCUSSION THREAD

**Discussion by masterwishx** [NationalSecurityAgency/ghidra#4615] [Q&A]
can find dark theme for ghidra .
using windows , if using Inverted colors , its somethink black but not really regular dark theme ...
is anyway to add it Ghidra ?

**Comment by masterwishx** [comment]
fixed particaly by changes colors in ghidra in inverted colors , but will be coll to have some dark themes in Ghidra natively

**Comment by astrelsky** [comment]
A dark theme for Ghidra is unnecessary and it is beautiful as is.

If it is too bright for you may I suggest turning on the lights in the room or perhaps investing in some sunglasses?

**Comment by dragonmacher** [comment]
The future is Dark.

**Reply by mumbel** [reply]
Dark and gloomy or yay my retinas

**Reply by dragonmacher** [reply]
Yes

![mystery](https://user-images.githubusercontent.com/48328597/197261974-afa5c652-83d0-4bf2-bd6f-fd94f6857c91.png)

**Comment by astrelsky** [comment]
> The future is Dark.

I prefer to see a bright future for Ghidra instead of a dark one.

**Reply by masterwishx** [reply]
> I prefer to see a bright future for Ghidra instead of a dark one.

its really fanny , i already made dark theme by changed colors in inverted mode , but for our eyes dark theme is much better and 
will be cool to have this like option to have and change some themes like in IDA maybe ? 

i changed color in decompiler and main window (listing) but dont have option in left side windows ...

**Comment by ekilmer** [comment]
@masterwishx Have you seen this PR? https://github.com/NationalSecurityAgency/ghidra/pull/4145, and it also links to this project https://github.com/zackelia/ghidra-dark

**Reply by masterwishx** [reply]
> Have you seen this PR? #4145, and it also links to this project https://github.com/zackelia/ghidra-dark

Thanks i tryed install in python  befor but have some errors .... i will check it again
